aspnetcore/mvc/advanced/custom-model-binding/samples/3.x/CustomModelBindingSample/Program.cs

```diff
@@ -0,0 +1,32 @@
+using CustomModelBindingSample.Data;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace CustomModelBindingSample
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            var host = CreateHostBuilder(args).Build();
+
+            using (var scope = host.Services.CreateScope())
+            {
+                var context = scope.ServiceProvider.GetRequiredService<AuthorContext>();
+
+                context.Authors.Add(new Author { Name = "Steve Smith", Twitter = "ardalis", GitHub = "ardalis", BlogUrl = "ardalis.com" });
+                context.SaveChanges();
+            }
```

Comment:
The 2.x sample takes care of this in `Startup.Configure`, but that's no longer "the norm". I could've pulled this out into its own method/class, but I think it's small enough to go here.

I left the author information here as it was. I can change it to someone else if you'd prefer, or even just fake it. There's no reference to these values in the topic itself.